apps/andi/lib/andi/event/event_handler.ex

```diff
@@ -50,9 +50,10 @@ defmodule Andi.Event.EventHandler do
     ingestion_update()
     |> add_event_count(author, data.id)
 
-    Ingestions.update_ingested_time(data.id, DateTime.utc_now())
+    data
```

Comment:
Saving twice to the database is unnecessary and error prone. The 2nd write would potentially overwrite the first if underlying implementation changed.

Just including the ingestion time as part of the single update...